webpack.config.js

```diff
@@ -0,0 +1,55 @@
+// Generated using webpack-cli https://github.com/webpack/webpack-cli
+
+const path = require("path");
+
+const isProduction = process.env.NODE_ENV == "production";
+
+const stylesHandler = "style-loader";
+
+const config = {
+  entry: "./src/index.ts",
+  output: {
+    path: path.resolve(__dirname, "static"),
```

Comment:
This should not go into static. We can move the build to `/dist`, and add it to `.gitignore`.

Reply:
Should I keep `/static` and move all generated javascript code into `/dist` or rename `/static` to `/dist`

Reply:
@shubham1172

Reply:
I missed this, static should stay as is. It should contain anything that does not get transpiled. `src` should contain typescript code. `dist` should contain the build result from `src`.